lib/services/elasticsearch.js

```diff
@@ -797,9 +805,11 @@ class ElasticSearch extends Service {
    * @return {Promise}
    */
   getMapping(request) {
-    const esRequest = initESRequest(request);
-
-    return this.esWrapper.getMapping(esRequest);
+    const
+      includeKuzzleMeta = request.input.args.includeKuzzleMeta,
+      esRequest = initESRequest(request);
+    console.log(includeKuzzleMeta)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)